sphinx/templates/quickstart/conf.py_t

```diff
@@ -76,32 +76,11 @@ exclude_patterns = [{{ exclude_patterns }}]
 
 # -- Options for HTML output -------------------------------------------------
 
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
-html_theme = 'alabaster'
```

Comment:
+0 to remove `html_theme` because I often changed this option after `sphinx-quickstart`.

Reply:
Okay, let's left not removed it.